config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3196,6 +3196,10 @@
 # syllabus.new.published.default=true
 
 ## TEST & QUIZZES (SAMIGO)
+# Minimum time between timer AJAX requests in miliseconds
+# DEFAULT: 5000
+# samigo.AjaxTimerMinReqScale=5000 
```

Comment:
can you make this camelCase like the other samigo config items?

Reply:
Yep, done.